.github/workflows/metricq-integration.yml

```diff
@@ -0,0 +1,69 @@
+name: MetricQ integration test
+
+on:
+  schedule:
+    - cron: "00 5,17 * * *"
+  workflow_dispatch:
+    inputs:
+      manager-ref:
+        description: git ref for metricq-manager
+        default: master
+        required: false
+      python-ref:
+        description: git ref for metricq-python
+        default: master
+        required: false
+
+jobs:
+  integrate:
+    runs-on: ubuntu-latest
+
+    services:
+      couchdb:
+        image: couchdb
+        env:
+          COUCHDB_USER: admin
+          COUCHDB_PASSWORD: admin
+      rabbitmq:
+        image: rabbitmq:alpine
+        env:
+          RABBITMQ_DEFAULT_USER: admin
+          RABBITMQ_DEFAULT_PASS: admin
+          RABBITMQ_DEFAULT_VHOST: /
+          RABBITMQ_ERLANG_COOKIE: Leibniz Butterkeks
+          RABBITMQ_NODENAME: rabbit@rabbitmq-server
+
+    steps:
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install Protoc
+        uses: arduino/setup-protoc@master
```

Comment:
Could you replace that with plain old apt-get? Nowadays, the action always gives some talk about rate limits.

See [#150](https://github.com/metricq/metricq-python/pull/150)